### DISCUSSION TITLE
Bind key on press rather than release

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#1149] [Q&A] [answered]
Is it possible to bind a key press rather than a key release?

I would like this to change volume.

**Comment by vaxerski** [comment, marked as answer]
all binds are by default on key press wdym

**Reply by ghost** [reply]
You are right.

How do I make hyprland call the keybinded command as long as the keyboard button is pressed down?

Currently it calls the command only one time. I want to call it as long as the key is pressed.

Basically I keybinding a volume increase. And I want to increase my volume by keeping the button pressed down, instead of repeatedly pressing it down multiple times.

**Reply by vaxerski** [reply]
https://wiki.hyprland.org/Configuring/Binds/#bind-flags

**Reply by ghost** [reply]
thank you, I missed that!

It works perfectly!
